changelogs/fragments/infinidat-removal.yml

```
@@ -0,0 +1,8 @@
+removed_features:
+  - "infini_export - has been moved to the ``infinidat.infinibox`` collection. A redirection is active."
```

```suggestion
  - "infini_* - all infinidat modules have been moved to the ``infinidat.infinibox`` collection. A redirection is active, which will be removed in version 2.0.0 (https://github.com/ansible-collections/community.general/pull/607)."
```
Please remove the other lines.

---

Done.